.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,12 @@
+# These templates are designed to encourage clear communication and efficient issue tracking, reflecting Intel's commitment to quality and customer focus
+blank_issues_enabled: false
+contact_links:
+  - name: Support
+    url: https://github.com/OpenVisualCloud/Media-Transport-Library/issues
+    about: If you have questions or need support, please visit our issue tracker.
+  - name: Documentation
+    url: https://github.com/OpenVisualCloud/Media-Transport-Library/wiki
```

I'd suggest using the docs link instead of an empty wiki.
```suggestion
    url: https://openvisualcloud.github.io/Media-Transport-Library/
```